plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -133,11 +133,11 @@ public PluginManager(Config cfg) {
     }
 
     private String getUserAgentInformation() {
-        String userAgentInformation= "Jenkins plugin manager/Dev";
```

Comment:
We don't know the version here, so we can just not provide one.

Reply:
Like any default version?